packages/eslint-plugin/docs/rules/object-curly-spacing.md

```diff
@@ -7,7 +12,7 @@ It adds support for TypeScript's object types.
 
 ## How to use
 
-```cjson
+```json
```

Comment:
this will make it look like an error on GH:
![image](https://user-images.githubusercontent.com/7462525/110027275-9ccdfe00-7ce6-11eb-8582-14eeaf9290e1.png)